src/lightning/pytorch/trainer/setup.py

```diff
@@ -184,8 +193,17 @@ def _log_device_info(trainer: "pl.Trainer") -> None:
     if TPUAccelerator.is_available() and not isinstance(trainer.accelerator, TPUAccelerator):
         rank_zero_warn("TPU available but not used. You can set it by doing `Trainer(accelerator='tpu')`.")
 
-    if _IPU_AVAILABLE and not isinstance(trainer.accelerator, IPUAccelerator):
-        rank_zero_warn("IPU available but not used. You can set it by doing `Trainer(accelerator='ipu')`.")
+    if _IPU_AVAILABLE:
+        if not _LIGHTNING_GRAPHCORE_AVAILABLE:
+            raise ModuleNotFoundError(
```

Comment:
I'm not sure this error should be added. Why would we block someone from using a IPU capable machine without graphcore? Don't these machines also have a regular CPU?

Reply:
Not as powerful as you would need for any regular task...

Reply:
lets address it together with HPU in #17465